ansible_collections/arista/avd/roles/eos_designs/templates/overlay/ebgp/address-family-evpn.j2

```diff
@@ -2,6 +2,17 @@
     peer_groups:
       {{ switch.bgp_peer_groups.evpn_overlay_peers.name }}:
         activate: true
+{% if switch.evpn_gateway is arista.avd.defined(true) %}
```

Comment:
Not sure we should use switch.evpn_gateway as the trigger for this since there may be mpls-related gateway stuff in the future that could need to coexist with vxlan gw functions on the same box (requiring 2 different peer groups). Would suggest to change to  {% if switch.evpn_gateway_evpn_l2 is arista.avd.defined(true) or switch.evpn_gateway_evpn_l3 is arista.avd.defined(true) %}.